web/models.py

```
@@ -213,6 +214,27 @@ def save(self, *args, **kwargs):
             self.slug = slugify(self.name)
         super().save(*args, **kwargs)
 
+    def add_fact(self, fact_text):
```

Ruff (0.8.2)

Missing return type annotation for public function `add_fact`

Add return type annotation: `bool`

(ANN201)